oidc/src/main/java/no/nav/common/oidc/auth/OidcAuthenticatorConfig.java

```diff
@@ -16,22 +17,25 @@
     // Client ID / Audience
     public String clientId;
 
-    // Name of the cookie where the id token should be stored
-    public String idTokenCookieName;
-
     // What type of user is being authenticated
     public IdentType identType;
 
-    // Name of the cookie where the refresh token should be stored (optional)
-    public String refreshTokenCookieName;
+    // Retrieves the id token from incoming requests
+    public TokenFinder idTokenFinder;
```

Comment:
Burde feltet hete `tokenFinder`? Siden det er en klasse `IdTokenFinder` som extender `TokenFinder`. Evt. finne nytt navn for `IdTokenFinder`

Reply:
Enig. Tar gjerne i mot forslag på navn istedenfor `IdTokenFinder` :p